### PR TITLE
Release v0.49.0

### PR DESCRIPTION
**Pull request by philipaconrad**
:information_source: This is the draft of our `v0.49.0` release notes. Comments and edit suggestions welcome! :writing_hand: 

This was a light month, so the release notes are much shorter than usual. :smile:

I didn't see any major features to call out, but @ashutosh-narkar might have some ideas for what to add if we want a callout at the top of the block.

### CODE REVIEW

**Comment by ashutosh-narkar** [comment]
Thanks for putting this together @philipaconrad!

**Comment by anderseknert** [comment]
Nice! Any chance we can have https://github.com/open-policy-agent/opa/pull/5576 included in v0.49.0?

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,46 @@
...
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.49.0
+
+This release focuses on bugfixes and documentation improvements, as well as a few small performance improvements.
+
+### Runtime, Tooling, SDK
+
+- runtime: Update rule index's trie node scalars to use custom map ([#5585](https://github.com/open-policy-agent/opa/issues/5585)) authored by @ashutosh-narkar reported by @alvarogomez93
```
Comment (srenatus) [inline CHANGELOG.md:12]: It would be nice to include why this was done, and what erroneous behaviour was fixed that way.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,46 @@
...
+- docs/graphql: non-nullable id argument and typo fix (authored by @philipaconrad)
+
+### Website + Ecosystem
+
+- Ecosystem:
+  - ccbr (authored by @niuzhi)
+
+- Website:
+  - Make warning banner closeable, edge is now blue (authored by @charlieegan3)
+  - Show prominent warning when viewing old docs (authored by @charlieegan3)
```
Comment (srenatus) [inline CHANGELOG.md:36]: I think these two first items can be combined.

**Review by srenatus — APPROVED** [review]
LGTM thanks for putting it together.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,46 @@
...
+- Update entrypoint documentation ([#5565](https://github.com/open-policy-agent/opa/issues/5565)) authored by @johanfylling reported by @robertgartman
+- Add missing folder argument in bundle build example (authored by @charlieegan3)
+- Clarify `crypto.x509.parse_certificates` docs (authored by @charlieegan3)
+- Added AWS S3 Web Identity Credentials info to tutorial (authored by @vishrana)
+- docs/graphql: non-nullable id argument and typo fix (authored by @philipaconrad)
+
+### Website + Ecosystem
+
+- Ecosystem:
+  - ccbr (authored by @niuzhi)
```
Comment (charlieegan3) [inline CHANGELOG.md:32]: Does it make sense to have PR links for all changes in the changelog? Worry that some might be missing context otherwise.

Comment (philipaconrad) [reply CHANGELOG.md:32]: I would have thought so! But apparently we've consistently stripped those from the CHANGELOG for many releases. I don't know the reasoning behind why, but I've kept the formatting to match what's there.

**Comment by philipaconrad** [comment]
@anderseknert Re: #5576, I can add that later this evening, or you can commit directly to this branch (I *think* you should have write access).

**Comment by philipaconrad** [comment]
Also, there's a weird Git error in the Go build targets; I don't have time before my flight to dig much further on it. If any of you all have time to figure out what's causing it, I'll do whatever PR cleanup is needed later this evening.

**Comment by charlieegan3** [comment]
Regarding #5576, there is still this comment that I'd like some input on:
https://github.com/open-policy-agent/opa/pull/5576#discussion_r1091593341. However, I can change that tomorrow quite quickly if need be.

**Review by ashutosh-narkar — APPROVED** [review]

**Comment by ashutosh-narkar** [comment]
@philipaconrad the notes lgtm. Can you add [this](https://github.com/open-policy-agent/opa/commit/16300529a2d80cc2d24239aeab6f8edad653145d) change to the notes and cut the release? Thanks!

**Comment by philipaconrad** [comment]
@ashutosh-narkar It is done. :+1:

I don't feel safe cutting the release tag until I know CI won't stop the image build/deploys for the docs/binaries. I'm going to wait until I either see things go green, or start dealing with the Git issue myself.

**Comment by philipaconrad** [comment]
CI failures seem related to [runner-images issue](this https://github.com/actions/runner-images/issues/6775)

**Comment by philipaconrad** [comment]
Everything *seems* okay now, except for the Windows build, which seems to have some weird issues going on related to permissions?

**Comment by ashutosh-narkar** [comment]
>Everything seems okay now, except for the Windows build, which seems to have some weird issues going on related to permissions?

@philipaconrad did you change something to make the other two previously failing checks to now pass?

**Comment by ashutosh-narkar** [comment]
For Windows the [following](https://github.com/open-policy-agent/opa/pull/5631/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52) seemed to work. Basically keeping `CI_GOLANG_DOCKER_MAKE` the same and moving `git config --global --add safe.directory /src` under the `ci-build-windows` target.

**Comment by ashutosh-narkar** [comment]
Closing in favor of https://github.com/open-policy-agent/opa/pull/5634.

**Pull request closed without merging**